lib/pwa.js

```diff
@@ -82,6 +83,53 @@ exports.list = function(skip, limit, sort) {
 };
 
 /**
+ * Add all PWAs from the DB into the text search index.
+ */
+exports.indexAllPwas = _ => {
```

Comment:
I'm wondering if we could extract this method go a PwaIndex class, and then just use it from here, as composition? we could also make the implementation more generic, but we might wait until it's needed.